.dockerignore

```
@@ -0,0 +1,4 @@
+.mk-libs
```

```suggestion
.mk-lib
```
I think this is a typo, right?

---

yes, you are right. Thanks.